packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -184,6 +186,9 @@ export class OdspDocumentService implements IDocumentService {
             concurrency,
             async (from, to, telemetryProps) => service.get(from, to, telemetryProps),
             async (from, to) => {
+                if (this.currentConnection !== undefined && !this.currentConnection.disposed) {
+                    this.currentConnection.requestOps(from, to);
```

Comment:
should this be behind some kind of flag?

Reply:
It feels pretty safe to me, with two caveats:
1. We have a chance to run with this change through ODSP stress test before it goes to prod. We will do it right away once it's in.
2. There is really nothing interesting here from client side (in terms of potential introduction of bugs), so introduction of policy/flag is more a kill switch for PUSH functionality. I'd rather have kill switch on PUSH side than on client here - if server disables this functionality, it's noop on client (and does not require code change on client).

@GaryWilber - any thoughts on state of # 2?

Reply:
We'll have a killswitch on the push side to disable processing get_ops requests - it would make it a no-op.

Reply:
@GaryWilber , do you have it implemented already? Or planning to add? This change will hit prod in a matter of 1-2 weeks after being approved

Reply:
The killswitch is implemented but it's not deployed to production yet. ETA to full prod deployment would be 2 weeks from now (8/16)